lib/ibanity/api/base_resource.rb

```diff
@@ -81,17 +88,17 @@ def setup_relationships(relationships, customer_access_token = nil)
         if relationship["data"]
           klass = Ibanity.const_get(Ibanity::Util.camelize(key))
           method_name = Ibanity::Util.underscore(key)
-          define_singleton_method(method_name) do
+          define_singleton_method(method_name) do |**query_params|
```

Comment:
do we need this if `query_params` isn't used?

Reply:
rigth